src/kernel/tty.zig

```diff
@@ -760,67 +684,117 @@ fn defaultAllTesting(p_i: u8, r: u8, c: u8) void {
     defaultVideoBufferTesting();
 }
 
+fn mock_entryColour(fg: u4, bg: u4) u8 {
```

Comment:
I remember there being some files under the test directory, could all of the mock functions go in there to keep this file cleaner?

Reply:
Sure, I just copies all the unit testing back here bc i couldn't get all of them working. I guess I can move all the public over to the testing file and keep the non-public in this file. What do you think?

Reply:
Sounds good :)